src/sections/image-with-text.liquid

```diff
@@ -0,0 +1,110 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add the "Image with text" section in the theme editor.
+
+  Theme Store required settings
+  - Image
+  - Image alignment: to align your image left or right
+
+  Theme Store optional settings
+  - Heading: title of the block
+  - Text: description of the block
+  - Button label: label associated with the button
+  - Button link: link of the button
+{%- endcomment -%}
+
+<div class="image-with-text">
```

Comment:
Should this parent wrapper be a `<section>` node?

Reply:
I agree.